src/dcrypto/dcrypto_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dcrypto
```

Comment:
What I mean by that is this
```shell
$ cat << ! > at.go
package at

func BlobAt() string {
  return hello()
}
```

```shell
$ cat << ! > at_test.go
package at

import (
  "testing"
)

func hello() string {
  return "hello"
}

func Test(t *testing.T) {
}
```

```shell
$ go test
```
will include `hello` from `at_test`

Where as if you made
```go
package at_test

import (
  "testing"
)

func hello() string {
  return "hello"
}

func Test(t *testing.T) {
}
```
you wouldn't be able to use unexported function `hello` from at_test.go in at.

It isn't a big deal nor show stopper, so all good.